services/node_delete.proto

```diff
@@ -36,9 +36,8 @@ option java_multiple_files = true;
  * - A deleted address book node SHALL be removed entirely from network state.
  * - A deleted address book node identifier SHALL NOT be reused.
  *
- * ### Record Stream Effects
- * Upon completion the "deleted" `node_id` SHALL be in the transaction
- * receipt.
+ * ### Block Stream Effects
+ * None.
```

Comment:
There will be StateChanges recorded here

Reply:
I haven't been specifically calling out state changes generally, as those are universal.

Do you find it valuable to state that explicitly?

Reply:
I wasn't sure what are being added here. Since it says None, I wanted to mention there are some block stream effects.

Reply:
Nothing being added, just removing specification for items that are _not_ in the block stream (the node ID is only published in receipt for the create transaction).